src/utils/metadata.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+import { getSetting } from 'docc-render/utils/theme-settings';
+import { absoluteURL } from 'docc-render/utils/assets';
+
+const themeTitle = getSetting(['meta', 'title'], process.env.VUE_APP_TITLE);
+const DEFAULT_DESCRIPTION = 'There\'s never been a better time to develop for Apple platforms.';
+const createMetaTags = ({ title, description, path }) => [
+  {
+    name: 'description',
+    content: description,
+  },
+  {
+    property: 'og:locale',
+    content: 'en_US',
+  },
+  {
+    property: 'og:site_name',
+    content: themeTitle,
+  },
+  {
+    property: 'og:type',
+    content: 'website',
+  },
+  {
+    property: 'og:image',
+    content: absoluteURL('/developer-og.jpg'),
```

Comment:
Definitely not a blocker, but it would be cool to work with the design team in the future to enhance these images to be a more visual representation of DocC pages instead of just the "Developer Documentation" text 🤩

Reply:
I totally agree. I would love to implement automated images for each page!